docs/source/parallelism.mdx

```diff
@@ -197,8 +197,9 @@ Implementations:
 - [Megatron-LM](https://github.com/NVIDIA/Megatron-LM) has an internal implementation - no API.
 - [Varuna](https://github.com/microsoft/varuna)
 - [SageMaker](https://arxiv.org/abs/2111.05972) - this is a proprietary solution that can only be used on AWS.
+- [OSLO](https://github.com/tunib-ai/oslo) - this is implemented based on the Hugging Face Transformers.
 
-🤗 Transformers status: as of this writing none of the models supports full-PP. GPT2 and T5 models have naive PP support. The main obstacle is being unable to convert the models to `nn.Sequential` and have all the inputs to be Tensors. This is because currently the models include many features that make the conversion very complicated, and will need to be removed to accomplish that.
+🤗 Transformers status: as of this writing none of the models supports full-PP. GPT2 and T5 models have naive inter-layer model parallelism support. The main obstacle is being unable to convert the models to `nn.Sequential` and have all the inputs to be Tensors. This is because currently the models include many features that make the conversion very complicated, and will need to be removed to accomplish that.
```

Comment:
I think using the term `PP` is not correct because the Transformers doesn't actually do pipelining.

Reply:
Do you suggest that because it doesn't do micro-batching and scheduling?

I'm fine with your proposed change, or we cold also leave it as is since it's a temp solution which can remove altogether once a proper PP is in place. Either way works for me.

Reply:
Both are fine. It's just suggenstion :)

Reply:
I reverted this :)